src/BPEssentials/Paths.cs

```diff
@@ -8,13 +8,14 @@ public class Paths
 
         public string SettingsFile { get; } = Path.Combine(EssentialsFolder, "settings.json");
 
+        public string WordBlacklistFile { get; } = Path.Combine(EssentialsFolder, "wordBlacklist.json");
+
```

Comment:
Wouldn't it be better if we do this in settings.json

Reply:
I rather not because it could be a very long file and settings have many other uses also settings already is messy af

Reply:
Then this File needs to be registerd in FileChecker.cs

Reply:
O